transifex-live-integration-main.php

```diff
@@ -200,6 +200,24 @@ static function do_plugin( $is_admin, $version ) {
 				}
 			}
 		}
+
+		add_action( 'wp_loaded', [ 'Transifex_Live_Integration', 'ping_tx' ] );
+	}
+
+	static function ping_tx() {
+		global $wp_version;
+		$settings = get_option( 'transifex_live_settings', array() );
+		if (isset($settings['api_key'])) {
+			$user_agent = array(
+				"transifex-live-wordpress-version/" . $wp_version,
+				"transifex-live-wordpress-plugin-version/" . TRANSIFEX_LIVE_INTEGRATION_PLUGIN_VERSION,
+				"transifex-live-wordpress-api-key/" . $settings['api_key']
+			);
+			wp_remote_get(
```

Comment:
Since it seems that we just want to log the call  why not use the more lightweight `wp_remote_head()` function?